README.md

```diff
@@ -1,26 +1,20 @@
 # BuyTogether
-
 Welcome to our group social project!.
-In this project we have developed a web application which suppose to help people group up
-and organize a large scale food buy.
-Doing so, we "cut" the amount of hands the product go through which will benefit the customer
+In this project, we have developed a web application that suppose to help people group up,
+and organize a large-scale food buy.
```

Comment:
not sure large-scale is correct here - a group of two is large scale ? maybe "... joint food purchase ... "